isis/src/control/objs/ControlNetVitals/ControlNetVitals.cpp

```diff
@@ -610,11 +610,13 @@ namespace Isis {
    */
   int ControlNetVitals::numImagesBelowMeasureThreshold(int num) {
     int count = 0;
-    foreach(int measureCount, m_imageMeasureCounts) {
-      if (measureCount > num) {
-        break;
+
+    QMap<int, int>::const_iterator i = m_imageMeasureCounts.constBegin();
+    while (i != m_imageMeasureCounts.constEnd()) {
+      if (i.key() < num ) {
+        count += i.value();
```

Comment:
QMap is always sorted, so you can just break if i.key() >= num

Reply:
:+1: